src/core/manager-serialize.c

```diff
@@ -158,6 +158,9 @@ int manager_serialize(
         (void) serialize_ratelimit(f, "dump-ratelimit", &m->dump_ratelimit);
         (void) serialize_ratelimit(f, "reload-reexec-ratelimit", &m->reload_reexec_ratelimit);
 
+        if (m->objective >= 0 && m->objective <= _MANAGER_OBJECTIVE_MAX)
```

Comment:
Hmm, why include _MANAGER_OBJECTIVE_MAX?

Reply:
Easy: because I misread and thought it was set to equal to the highest actual value